client/components/tooltip/tooltip-base.tsx

```diff
@@ -14,12 +14,35 @@ import './style.scss';
 const rootElement =
 	document.getElementById( 'wpbody-content' ) || document.body;
 
-const isEventTriggeredWithin = ( event, element ) =>
-	element && ( element === event.target || element.contains( event.target ) );
+const isEventTriggeredWithin = (
+	event: MouseEvent,
+	element?: ChildNode | null
+) => {
+	if ( ! element ) {
+		return false;
+	}
+	if ( element === event.target ) {
+		return true;
+	}
+	if ( event.target instanceof Node && element.contains( event.target ) ) {
+		return true;
+	}
+};
```

Comment:
We might want to add a final `return false` to this function, for a consistent `boolean` return type.

Reply:
Resolved in 967889909220595bc77cdbc9165d80cd421029ec